dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/SelectGuestbookPage.java

```diff
@@ -77,25 +77,8 @@ public Guestbook getSelectedGuestbook() {
 
     // -------------------- LOGIC --------------------
     public String init() {
-        if (persistentId != null) {
-            dataset = datasetService.findByGlobalId(persistentId);
-        } else if (datasetId != null) {
-            dataset = datasetService.find(datasetId);
-        }
-
-        if (dataset == null) {
-            return permissionsWrapper.notFound();
-        }
-
-        // Check permisisons
-        if (!permissionsWrapper.canUpdateDataset(dvRequestService.getDataverseRequest(), dataset)) {
-            return permissionsWrapper.notAuthorized();
-        }
-        if (!dataset.getLocks().isEmpty()) {
-            return permissionsWrapper.notAuthorized();
-        }
```

Comment:
Why we removed this code? Now everyone can access this page. And you will get null pointer exceptions when rendering page.

Reply:
My bad. I've pushed that by mistake instead of what i wanted to.